package-lock.json

```diff
@@ -1,25 +1,52 @@
 {
   "name": "@asyncapi/specs",
   "version": "5.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 3,
```

Comment:
Just letting you be aware this is changing from lockfileVersion 1 to 3. We are covered by CI so I would say there is nothing to be concerned about.

Reply:
> Just letting you be aware this is changing from lockfileVersion 1 to 3.

Okay, I see...

> We are covered by CI so I would say there is nothing to be concerned about.

That's good to know! :smile: